src/components/core/core.js

```diff
@@ -349,6 +349,11 @@ export default class Core extends UIObject {
     }
   }
 
+  appendToParent() {
+    let hasCoreParent = this.$el.parent() && this.$el.parent().length
```

Comment:
Is there any possibility of `$el` or `parent()` return `undefined`?

Reply:
`$el` is valid just after the `UIObject` constructor is called, so it would be `undefined` only if some rogue component destroys the element. The `parent()` call can return undefined if the core element is removed from DOM or `destroy`ed.

Reply:
I think it's safe to assume that $el is always valid.